.github/workflows/push-main.yml

```diff
@@ -71,6 +71,7 @@ jobs:
         uses: google-github-actions/auth@v0.8.0
         with:
           token_format: "access_token"
+          access_token_lifetime: "10800s"
```

Comment:
All builds took within 3 hours. It looks fine to me.
https://github.com/gitpod-io/workspace-images/actions/workflows/push-main.yml